swayfmt/tests/mod.rs

```diff
@@ -1602,3 +1602,61 @@ impl MyContract for Contract {
 "#,
     );
 }
```

Comment:
We should add a few test cases for each of the bugs fixed (under `swayfmt/src/items/<item type>/tests.rs`)